conf/messages.en

```diff
@@ -251,20 +251,20 @@ page.iht.registration.checklist.title = Before you start registration
 page.iht.registration.checklist.label1 = You’ll need to have all of the details listed below before you start.
 page.iht.registration.checklist.label2 = There’s no way to save your progress until you have given all of these details, so only start once you have everything you’ll need.
 page.iht.registration.checklist.label3 = It should take you no more than 30 minutes to finish all of registration.
-page.iht.registration.checklist.deceased.title = The deceased is the person who has died. You’ll need their:
+page.iht.registration.checklist.deceased.title = You’ll need the person who has died’s:
 page.iht.registration.checklist.deceased.label3 = date of death
 page.iht.registration.checklist.deceased.label5 = last known contact address
-page.iht.registration.checklist.deceased.label6 = country where they had their permanent home – you can only use this service if the deceased lived in England or Wales
+page.iht.registration.checklist.deceased.label6 = country where they had their permanent home – you can only use this service if the person who has died lived in England or Wales
 page.iht.registration.checklist.deceased.label7 = relationship status
-page.iht.registration.checklist.deceased.reveal.label1 = The deceased’s first and last name will be on the death certificate. If you didn’t know the deceased, ask their family or friends for some details if you don’t have them.
+page.iht.registration.checklist.deceased.reveal.label1 = The person who has died’s first and last name will be on the death certificate. If you didn’t know the person who has died, ask their family or friends for some details if you don’t have them.
 page.iht.registration.checklist.deceased.reveal.label2 = The last known contact address is where they had their post sent before they died. This could be to a home that they owned or where they last lived.
```

Comment:
The content "If you didn’t know the deceased, ask their family or friends for some details if you don’t have them." is still there.